test/cql-pytest/test_describe.py

```diff
@@ -733,6 +733,49 @@ def test_table_options_quoting(cql, test_keyspace):
     finally:
         cql.execute(f"DROP TYPE {test_keyspace}.\"{type_name}\"")
 
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #16317")]), "vnodes"],
+                         indirect=True)
+def test_hide_cdc_table(scylla_only, cql, test_keyspace):
+    cdc_table_suffix = "_scylla_cdc_log"
+    
+    with new_test_table(cql, test_keyspace, "a int primary key, b int", "WITH cdc = {'enabled': true}") as t:
+        t_name = t.split('.')[1]
+        cdc_log_name = t_name + cdc_table_suffix
+
+        # Check 't_scylla_cdc_log' is hidden in DESC TABLES
+        desc_tables = cql.execute("DESC TABLES")
+        assert cdc_log_name not in [r.name for r in desc_tables]
+        cdc_log_table_entry = cql.execute(f"SELECT * FROM system_schema.tables WHERE keyspace_name='{test_keyspace}' AND table_name='{cdc_log_name}'").all()
+        assert len(cdc_log_table_entry) == 1 # assert the table exists
+
+        # Check 't_scylla_cdc_log' is hidden in DESC KEYSPACE ks
```

Comment:
Please say something in the comment about the ALTER TABLE case. Apparently it's not completely hidden? Also, what is that "or" condition - which of the two actually happens in this test?

Reply:
Described briefly the test in a comment above it.

The `or` was there because statement like `DESC KEYSPACE ks` contains one row for base table with CREATE statement and one row for log table with ALTER statement.
But I've switched it to check if current row is a row for log table.